### DISCUSSION TITLE
A-S Pressure Sensor Reads Zero, Blue LED On, Flashing Green When Reading

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#1010] [Q&A]
A few days ago, the A-S Pressure Sensor began reading zero nearly all of the time.  In "normal" pool mode, the Intelliflow VS pump runs at 1750 RPM, 16-18 GPM, and the pressure for a clean sand filter is in the range of 8.5-11.5 PSI.  When the Polaris cleaner is running, the pump is  at 2650 RPM and the reported pressure has been 18-20 PSI.

It's now reporting zero during normal pool mode and 2-3 PSI when the Polaris is running.

I tried calibrating the probe early in the morning, shortly before the pool turns on for the day (the pool is "On" from 8:00 AM to 8:00 PM).

The LEDs indicate the sensor is in I2C mode (Blue LED, flashing green when reading pressure).

The pressure sensor is about 8 months old &  is connected to a WhiteBox T3 HAT.  I disconnected the probe & reconnected it fully, just in case the connector had loosened to no avail.  I backwashed the filter last week in case something nasty was interfering with the sensor.  Although the water flow rate increased slightly, the pressure readings did not change from zero.

Do you have any ideas what I  can do to get the pressure readings back to normal?  Should I reset the probe or does that have bad consequences?

Thanks for your help & suggestions!!

**Comment by rstrouse** [comment]
Please inspect the log to see if it is actually reporting the pressure.   Also how long is the wire you are using?  Did you lengthen it?

**Reply by FlaMike** [reply]
I've not checked the log yet, but can do so.  I believe the sensor is reporting something because when the Polaris runs I do get a reading of around 3 PSI, then it drops  to zero once the pump slows to "normal" filtering speed.

I did lengthen the cable when it was first installed using the A-S extension kit, back in December or January.  I've not seen any damage to the cable or the shrink wrap they provided with the kit, however.

Nixie and REM report identical readings, for what that's worth.  About to generate a log to examine...

**Reply by FlaMike** [reply]
I am not certain where to look for pressure data in the logs.  I'm attaching a brief replay file.  I cleared messages prior to the capture.  The pressure sensor is configured in REM to get a reading every 10 seconds (10,000 ms).  Am I looking the correct place?

Thanks!

Ooops.  replay is down 2 messages.  Sorry!!!

**Reply by FlaMike** [reply]
I noticed in REM I  configured the pressure sensor to report only when changed.  Just unchecked the box & will create a new capture.

**Reply by FlaMike** [reply]
Still not seeing "pressure" in the logs I have...

**Comment by FlaMike** [comment]
[replay (11).zip](https://github.com/user-attachments/files/16461635/replay.11.zip)

**Reply by FlaMike** [reply]
For what it's worth (possibly not much) I logged onto Nixie via SSH and entered  the following command:  i2cget 1 0x6a.  The i2c bus is '1' and the sensor address is 0x6a.

The response (after the confirmer) was 0xff.  I assume that's the reported pressure value from the sensor?  If so, it  seems pegged (low?) and both REM and Nixie would be accurate the in the reported value of 0 PSI.

Does this make sense?  If so, is it reasonable to assume the probe is toast?

Thanks!  Per usual, I am groping the dark here.

**Reply by rstrouse** [reply]
look a the logs in REM.  If you are using pm2 use pm2 monit then select REM.  You should see the pressure sensor being polled.  It is possible that the sensor is toast.  You can try to reset the device using the reset button on the PRS device.  This should factory reset the device in case there is something wrong with it memory wise.

**Reply by FlaMike** [reply]
Here is  what I get via pm2 monit  Time marches along but otherwise the content doesn't seem to change much.

![image](https://github.com/user-attachments/assets/32f6556c-5f7c-4f11-a418-a1ed19a3f206)

If I do a reset via REM on the probe, do I need to put it back into I2C mode?

Thanks for your help!

**Reply by rstrouse** [reply]
Resetting the probe should not put it into serial mode.

EDIT:  Expand the width of that window so you can see the messages.

**Comment by FlaMike** [comment]
Thank you for your quick & helpful response!!  I will reset the probe in
the morning before the pool starts & then recalibrate it.

Hoping to post good news in the morning!

**Reply by FlaMike** [reply]
I did a reset followed by a calibrate via REM this morning after the pool had been off for hours.  The pool turned on about 1/2 hour ago & the pressure reading is 10. PSI, a perfectly reasonable reading based on the 6 months or so since installing the pressure probe.  :-)

And it's 21 PSI when the Polaris is running!
